### PR TITLE
Update README with details for newbies...

### PR DESCRIPTION
**Pull request by danielpunkass**
Baesd on my experience getting started with the project today, hopefully get people up and going faster hwen they are less familiar with node, npm, and the high-level architecture of the scrapers. This is a clean resubmission of #220 which I managed to screw up and got messy.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +62,22 @@ To scrape data from a site, you either need to:
...
+      availability: {},
+      timestamp: 2021-04-07T18:27:37.498Z,
+      siteTimestamp: 2021-04-07T17:23:33.046Z,
+      signUpLink: 'https://www.cvs.com/immunizations/covid-19-vaccine?icid=cvs-home-hero1-banner-1-link2-coronavirus-vaccine',
+      latitude: 42.6757741,
+      longitude: -71.12855859999999
+    },
+
+Some values such as timestamp will be supplied by default, so you only need to supply them if overriding a default value.
+
```
Comment (harcod) [inline README.md:80]: `timestamp` is not supplied by default.
`latitude` and `longitude` are computed if there is a valid `street`, `city`, and `zip`

**Review by harcod — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (danielpunkass) [reply README.md]: Thanks for catching that. I must have gotten the wrong impression about timestamp during development. I've submitted an update that rephrases the whole line.

**Review by harcod — APPROVED** [review]
LGTM! Thanks.

**Comment by harcod** [comment]
All PRs require two approving reviews.  You can nudge others to review your PR in the #dev channel on Slack

**Comment by danielpunkass** [comment]
@harcod Thanks! Could you share information about how to join the Slack? I don't see any details on the README. Am I missing an important contributors page? I didn't see any info about a Slack on the macovidvaccines Contributors.md either.

**Comment by harcod** [comment]
Send an email to macovidvaccines@gmail.com

**Review by iann — APPROVED** [review]
Looks good!

**Comment by iann** [comment]
Thank you!

**Pull request merged**